diff --git a/tests/integration/tests.mk b/tests/integration/tests.mk
index ba4eeed33c..5c8f792dd6 100644
--- a/tests/integration/tests.mk
+++ b/tests/integration/tests.mk
@@ -116,7 +116,7 @@ test.integration.kube: test.integration.kube.presubmit
 # Presubmit integration tests targeting Kubernetes environment. Really used for postsubmit on different k8s versions.
 .PHONY: test.integration.kube.presubmit
 test.integration.kube.presubmit: | $(JUNIT_REPORT) check-go-tag
-	$(RUN_TEST) ./tests/integration/... -timeout 30m \
+	$(RUN_TEST) ./tests/integration/pilot/... -timeout 30m -run ^TestTraffic \
 	${_INTEGRATION_TEST_FLAGS} ${_INTEGRATION_TEST_SELECT_FLAGS} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
